README.md

```diff
@@ -31,3 +31,82 @@ The scenario-explorer-portalconfig can optionally be checked out with the `local
 ```sh
 docker compose up --build
 ```
+
+## Keycloak
+
+To configure a custom Keycloak login Theme, follow these steps in the Keycloak Admin Console at `https://<your-portal-domain>/auth`.
+
+### 1. Realm Selection
+
+After logging in to the Keycloak Admin Console, switch to the Keycloak Realm named `UrbanModelPlatform`.
+
+### 2. Login Tab
+
+Navigate to `Realm Settings` and select the `Login` tab. Configure the following settings for optimal login experience:
+
+| Setting               | Enabled/Disabled |
+|-----------------------|------------------|
+| User Registration     | On               |
+| Forgot Password       | On               |
+| Remember Me           | Off              |
+| Email as Username     | On               |
+| Login with Email      | On               |
+| Duplicate Emails      | Off              |
+| Verify Email          | On               |
+| Edit Username         | Off              |
+
+**Details:**
```

Comment:
Can we also provide a link to the corresponding Keycloak documentation?

Reply:
Very good point! A link is now added.